File: Difficulty: Easy/Reverse an Array/reverse-an-array.py
class Solution:
    def reverseArray(self, arr):
        # code here
        l =0
        r = len(arr) -1
        
        while(l<r):
            temp = arr[l]
            arr[l] = arr[r]
            arr[r] = temp
            l+=1
            r-=1
        
        

File: Difficulty: Easy/Reverse an Array/reverse-an-array.java
class Solution {
    int[] swap(int[] arr, int i, int j){
        if(i>j) return arr;
        int temp = arr[i];
        arr[i] = arr[j];
        arr[j] = temp;
        
        return swap(arr, i+1, j-1);
    }
    
    public void reverseArray(int arr[]) {
        // code here
        swap(arr, 0, arr.length-1);
        
    }
}